### DISCUSSION TITLE
NARR forcing grid won't load, LDT process dies

### DISCUSSION THREAD

**Discussion by dechowjack** [NASA-LIS/LISF#1163] [Q&A]
Hi,

I am attempting to prepare a LIS input file using NARR met forcings as opposed to NLDAS2, I have been successfully able to run large domains using NLDAS but when attempting to switch to NARR I am running into some issues. I have provided a NARR forcing directory (/discover/nobackup/jdechow/LISF/ldt/metforcing/narr) but my runs seem to die when trying to load in the NARR forcing grid. Here are two errors I am seeing, the first from the slurm file and the second from the ldtlog

`srun: cluster configuration lacks support for cpu binding
 set the number of forcing variables in init_narr
 stopping..
____________`

` - - - - - - - - - Noah-MP Parameters - - - - - - - - - - - -
  - - - - - - - - Meteorological Parameters - - - - - - - -
 MSG: Initializing NARR forcing grid ... `

At this point the process is killed. Any advice for working with NARR for the first time, or some sneaky lines I need to add? I have been scouring the LDT and LIS user guides but I'm really coming up blank. Any advice is appreciated greatly!!

Jack

**Comment by dmocko** [comment]
Hi @dechowjack 

Is there any reason you need to use the NARR metforcing dataset?

This metforcing dataset may not have been run for sometime, and I'm not sure if it's still working or not.

Are you using the GRIB-1 NARR data that is in this directory?   /discover/nobackup/projects/lis/MET_FORCING/NARR

Please let us know your desired domain and years of interest for your simulation, and we can recommend a different forcing.

**Reply by dechowjack** [reply]
Hi @dmocko 

We are working in Alaska for WY22 and had determined that NARR would work best for our simulations, but are open to other suggestions. Also, no, I am not using the NARR data in that directory but I will try that next!

**Reply by dmocko** [reply]
The NARR data in that directory is not updated to the current year, but the format of that data should be what works with the NARR reader.  If you are trying to work with a different format NARR data, that could be why the code wasn't working for you.

This paper used three different forcing datasets over North America, and these forcings include coverage over Alaska:
https://doi.org/10.5194/tc-15-771-2021

The GDAS data is updated in real-time also on Discover:
/discover/nobackup/projects/lis/MET_FORCING/GDAS

The MERRA-2 data has a couple month of lag, but the latest is also on Discover.

The ECMWF forcing is also on Discover, but it may be restricted within our group.
